### DISCUSSION TITLE
Add a navigation tool bar

### DISCUSSION THREAD

**Discussion by alexec** [community/community#16047] [Pull Requests]
It is impossible to get directly from a check within GitHub Actions to the PR that caused the check. Yet this is the most common action I wish to take. Please add a “Back to Pull Request” link at the top of the page. This link should always be visible, i.e. if the page scrolls I can still see it.

**Comment by jithunnair-amd** [comment]
I agree. It is quite annoying frankly to see the title of the PR on the Actions page but no link to get to the PR that launched the action.
